ml_metadata/metadata_store/metadata_store_server_main.cc

```diff
@@ -328,6 +355,23 @@ DEFINE_string(mysql_config_user, "",
               "The mysql user name to use (Optional parameter)");
 DEFINE_string(mysql_config_password, "",
               "The mysql user password to use (Optional parameter)");
+DEFINE_string(mysql_config_sslcert, "",
+              "This parameter specifies the file name of the client SSL certificate.");
+DEFINE_string(mysql_config_sslkey, "",
+              "This parameter specifies the location for the secret key used for the "
+              "client certificate.");
+DEFINE_string(mysql_config_sslrootcert, "",
+              "This parameter specifies the name of a file containing SSL "
+              "certificate authority (CA) certificate(s).");
+DEFINE_string(mysql_config_sslcapath, "",
+              "This parameter specifies path name of the directory "
+              "that contains trusted SSL CA certificates.");
+DEFINE_string(mysql_config_sslcipher, "",
+              "This parameter specifies the list of permissible ciphers for "
+              "SSL encryption.");
+DEFINE_bool(mysql_config_verify_server_cert, false,
+              "This parameter enables verification of the server certificate "
+              " against the host name used when connecting to the server.");
```

Comment:
```suggestion
DEFINE_bool(mysql_config_verify_server_cert, false,
              "When certificates are provided, this parameter enables verification of the server certificate "
              " against the host name used when connecting to the server.");
```

I know we discussed this, just re-reporting for archival purposes.